remotion/compositions/templates/talk/talks.types.ts

```diff
@@ -1,16 +1,19 @@
 import {zColor} from '@remotion/zod-types';
 import {z} from 'zod';
 
+export const TalkBrandedSpeakerSchema = z.object({
+	pictureUrl: z.string(),
+	name: z.string(),
+	company: z.string().optional(),
+	job: z.string().optional(),
+})
+
 export const TalkBrandedSchema = z.object({
 	backgroundColor: zColor(),
 	title: z.string(),
 	startingDate: z.date(),
 	location: z.string().optional(),
 	logoUrl: z.string(),
-	speaker: z.object({
-		pictureUrl: z.string(),
-		name: z.string(),
-		company: z.string().optional(),
-		job: z.string().optional(),
-	}),
+	speaker: z.optional(TalkBrandedSpeakerSchema),
+	speakers: z.optional(z.array(TalkBrandedSpeakerSchema)),
```

Comment:
```suggestion
	speakers: z.optional(z.array(TalkBrandedSpeakerSchema)),
```

Tu peux te permettre un breaking change ici :)